### ISSUE TITLE
[Linux][aarch64] discord-rpc causes crashing

### ISSUE THREAD

**Issue by theofficialgman** [KRTirtho/spotube#1649]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

spotube (arm64 deb) opens with a totally empty window

### Expected Behavior

spotube works normally

### Steps to reproduce

I opened the app

### Operating System

Linux Ubuntu Noble 24.04 ARM64

### Spotube version

v3.7.1

### Installation source

GitHub Releases (Binary)

### Additional information

```
gman@gman-nob:/tmp$ spotube 
package:media_kit_libs_linux registered.
flutter: media_kit: WARNING: package:media_kit_native_event_loop not found.
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: Invalid argument(s): Failed to load dynamic library '/usr/share/spotube/lib/libdiscord-rpc.so': /usr/share/spotube/lib/libdiscord-rpc.so: cannot open shared object file: No such file or directory
#0      _open (dart:ffi-patch/ffi_dynamic_library_patch.dart:11)
#1      new DynamicLibrary.open (dart:ffi-patch/ffi_dynamic_library_patch.dart:22)
#2      DiscordRPC.initialize (package:dart_discord_rpc/dart_discord_rpc_native.dart:152)
#3      main (package:spotube/main.dart:74)
<asynchronous suspension>


```

```
gman@gman-nob:/usr/share/spotube/lib$ file *
libapp.so:                                 ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[md5/uuid]=9bc2f994788b799ae0ddad13a361db75, stripped
libdart_discord_rpc_plugin.so:             ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=31d323d0bb9fe65dd8351acbb9e844acf560315d, not stripped
libdiscord-rpc.so:                         ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, with debug_info, not stripped
libfile_selector_linux_plugin.so:          ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=ece3fdaf0eed709f4fb8882277e47e8d2cff5bcd, not stripped
libflutter_linux_gtk.so:                   ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=3d128f4c333348b1a0b5e1c13072a8844d325f86, stripped
libflutter_secure_storage_linux_plugin.so: ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=2dca55350c7ec1225fd9bb0dff270db056a674a9, not stripped
libgtk_plugin.so:                          ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=3c5bffb558c7b9f97cfc101baa1524291e848272, not stripped
liblocal_notifier_plugin.so:               ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=381453f1198879bb7df3edcedbc24d1f47ccbd39, not stripped
libmedia_kit_libs_linux_plugin.so:         ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=32ced4640c206fd8d31e390af56554d14c5c7e0b, not stripped
libmedia_kit_native_event_loop.so:         ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=d8d3d04683f3d1b4ff4fa500846cf0aa03944e37, not stripped
libmetadata_god.so:                        ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, with debug_info, not stripped
libscreen_retriever_plugin.so:             ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=a8297326483ae62cb33560424d8a3de6b35f1002, not stripped
libsystem_theme_plugin.so:                 ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=6d8b2c55f3ca45c657b3b19f8837944df1321322, not stripped
libsystem_tray_plugin.so:                  ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=6fb10c6fe488f6a510e78213204f88118f15f0cf, not stripped
libtray_manager_plugin.so:                 ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=aab7b85e8858bf527fe428d65cdee231a43ad184, not stripped
liburl_launcher_linux_plugin.so:           ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=62bd7be6380b82be122279e343a612160f5b0c04, not stripped
libwindow_manager_plugin.so:               ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=927d23e2f2b63357c5bbb5ae88b6e265aec61ab3, not stripped
```

https://github.com/KRTirtho/spotube/issues/715

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by theofficialgman** [comment]
same issue on x86_64 if I move or remove the `libdiscord-rpc.so` binary. the lack of this binary is preventing the application from initializing

**Comment by KRTirtho** [comment]
It's been fixed in 3.8.0. Now metadata_god, flutter_discord_rpc etc plugins are built from source so now we've aarch64 support

**Comment by KRTirtho** [comment]
Try out 3.8.0
